avaframe/in1Data/getInput.py

```diff
@@ -497,3 +504,169 @@ def fetchReleaseFile(inputSimFiles, releaseScenario, cfgSim, releaseList):
+    dem = IOf.readRaster(inputData['demFile'], noDataToNan=True)
+    dem['originalHeader'] = dem['header'].copy()
+    methodMeshNormal = cfg.getfloat('GENERAL', 'methodMeshNormal')
+    # get normal vector of the grid mesh
+    dem = DFAtls.getNormalMesh(dem, methodMeshNormal)
+    dem = DFAtls.getAreaMesh(dem, methodMeshNormal)
+    header = dem['header']
+    csz = header['cellsize']
+
+    # define parameters to be saved
+    relFeatures = []
+    relMaxZ = []
+    relMinZ = []
+    relSlope = []
+    relArea = []
+    relAreaP = []
+    relPath = pathlib.Path(avaDir, 'Outputs', 'com1DFA', 'releaseInfoFiles')
+    fU.makeADir(relPath)
+    relDFDict = {}
+    # loop over all relFiles
+    for relFile in relFiles:
+        # for relF in relFiles:
+        releaseLine = {}
+        releaseLine = shpConv.readLine(relFile, 'release1', dem)
+        releaseLine['file'] = relFile
+        releaseLine['type'] = 'Release'
+        releaseLine['thicknessSource'] = ['artificial'] * len(releaseLine['id'])
+        releaseLine['thickness'] = ['1.'] * len(releaseLine['id'])
+        # convert release line to a raster with 1 set inside of the release line
+        # and compute projected and actual areas
+        areaActualList, areaProjectedList, releaseLine = computeAreasFromRasterAndLine(releaseLine, dem)
+        # compute projected areas using shapely
+        projectedAreas = computeAreasFromLines(releaseLine)
+
+        # create dict for raster data for each feature
+        for index, relRaster in enumerate(releaseLine['rasterData']):
+            zArray = np.where(relRaster > 0, dem['rasterData'], np.nan)
+
+            # compute slope of release area
+            _, _, NzNormed = DFAtls.normalize(dem['Nx'], dem['Ny'], dem['Nz'])
+            nzArray = np.where(relRaster > 0, NzNormed, np.nan)
+            slopeArray = np.rad2deg(np.arccos(nzArray))
+            meanSlope = np.nanmean(slopeArray)
+
+            # compute elevation stats
+            zMax = np.nanmax(zArray)
+            zMean = np.nanmean(zArray)
+            zMin = np.nanmin(zArray)
+
+            # create a dataframe
+            relFeatures.append(releaseLine['Name'][index])
+            relMaxZ.append(zMax)
+            relMinZ.append(zMin)
+            relSlope.append(meanSlope)
+            relAreaP.append(areaProjectedList[index] / 10000.)
+            relArea.append(areaActualList[index] / 10000.)
+
+            # print info
+            log.info('++++ Release feature %s ++++++++++' % releaseLine['Name'][index])
+            log.info('release area projected: %.2f' % areaProjectedList[index])
+            log.info('release area projected using shapely: %.2f' % projectedAreas[index])
+            log.info('release area real: %.2f' % areaActualList[index])
+            log.info('maximum elevation: %.2f' % zMax)
+            log.info('mean elevation: %.2f' % zMean)
+            log.info('minimum elevation: %.2f' % zMin)
+            log.info('mean slope: %.2f' % meanSlope)
```

Comment:
To keep the clatter in the log to a minimum, I think this should be dropped to debug. Especially since this information is saved and easily readable info about it is given in the log.info below...

Reply:
I think it is the only output of the runScript.. but for example all the parameters of the com1DFA config are listed in the log although only the methodMeshNormal is used - maybe we could decrease this info in the log, but I think it is quite nice to see the stats printed and not have to open the file - but I moved it to debug